### ISSUE TITLE
Abnormal High Loss and Erratic Robot Behavior when Fine-Tuning pi05_baseCompared to pi0_base

### ISSUE THREAD

**Issue by DYwasd** [huggingface/lerobot#2255]
Hello! I encountered a significant difference and issue when fine-tuning the pi0_baseand pi05_basepolicies on the same dishwasherdataset using lerobot. I would appreciate some guidance.

### Training Command
The command I used for pi05_basefine-tuning is below (the command for pi0_basewas similar, only changing the policy.typeand policy.repo_id/pretrained_path):
```powershell
python src/lerobot/scripts/lerobot_train.py \
    --dataset.repo_id=dishwasher \
    --policy.type=pi05 \
    --output_dir=./outputs/pi05_training \
    --job_name=pi05_training \
    --policy.repo_id=lerobot/pi05_base \
    --policy.pretrained_path=lerobot/pi05_base \
    --policy.compile_model=true \
    --policy.gradient_checkpointing=true \
    --policy.dtype=bfloat16 \
    --steps=6000 \
    --num_workers=32 \
    --policy.scheduler_decay_steps=3000 \
    --policy.device=cuda \
    --batch_size=32 \
    --log_freq=20 \
    --save_freq=2000
``` 

### Experimental Setup & Observations
**​​1. Fine-tuning pi0_base:​​**
- Initial Loss: ~0.6
- Final Loss: Reduced to ~0.03
- Model Behavior: The trained model performs normally and is usable.

**2. Fine-tuning pi05_base:​​**
- Initial Loss: Very high, ~40,000+
- Final Loss: Reduced to ~100
- Model Behavior: ​​The Problem!​​ When deploying the trained model to the ALOHA robot, the robot's movements are ​​very erratic​​ and it fails to complete the dishwasher task.

### Core Questions
1. Is the high Loss value (40k+ -> 100) during pi05_basefine-tuning normal? If so, why is the Loss scale so different (by orders of magnitude) from pi0_base?
2. Could the high Loss value be the cause of the model's erratic behavior during real-world deployment?
3. Are there specific training techniques, hyperparameter adjustments, or data preprocessing requirements for the pi05model?
4. Besides the Loss value, what other metrics should I check to diagnose the pi05model's issues?

**Comment by pkooij** [comment]
Thank you for submitting this. @DYwasd Your loss for Pi0.5 seems very high. May I also ask if you are training on a Nvidia gpu or MacBook? Could you also maybe check if the initial loss is also that high for the pretrained 'pi05_libero' checkpoint?

**Comment by DYwasd** [comment]
> Thank you for submitting this. [@DYwasd](https://github.com/DYwasd) Your loss for Pi0.5 seems very high. May I also ask if you are training on a Nvidia gpu or MacBook? Could you also maybe check if the initial loss is also that high for the pretrained 'pi05_libero' checkpoint?

Thanks for your reply and the suggestions. @pkooij 
​​**Training Hardware:​​** I'm training on an ​​NVIDIA GPU (A6000)​​.
**​​Initial Loss Check:​​** I verified the initial loss using the provided pretrained checkpoint pi05_libero, and it ​​remains very high​​, starting at ​​over 44,000​​,as shown in the screenshot below​​.
<img width="1679" height="311" alt="Image" src="https://github.com/user-attachments/assets/48040d2c-f57b-416d-8ba5-40867c9175eb" />
The attached files contain partial training logs for pi0and pi05 for your reference.
[pi05.log](https://github.com/user-attachments/files/23043938/pi05.log)
[pi0.log](https://github.com/user-attachments/files/23043939/pi0.log)
